src/components/ProductPage/BodySection/BodySection.tsx

```diff
@@ -63,11 +69,11 @@ export const BodySection = ({ section, images }: { section: SectionProps; images
           })}
         </div>
       )}
-      {section.callToAction ? (
+      {section.callToAction && (
```

Comment:
curios why this is change? should it return null if doesnt exist? to prevents potential errors or rendering issues that could occur

Reply:
I don't see a downside to this check as `FeatureLink` depends on a lot of stuff from `section.callToAction` - the component may be able to handle that being missing but given the purpose of the PR I'm guessing it doesn't.

As for returning null, I revisited the React docs on this. [The latest on conditional rendering](https://react.dev/learn/conditional-rendering#logical-and-operator-) doesn't seem to have a problem with using this versus a ternary that returns `null` in the false case (which is what I've recommended to do in the past) - equating returning `false` as equivalent to `null` in the JSX rendering tree. In the past, there was a performance/memory benefit to `null` over `false` - maybe there still is.

So basically we can use either.

Reply:
it'll do the same thing i believe, but this uses a shorter syntax, if `section.callToAction` is true, render the `FeatureLink` component

(if this part is true) && (this part will render)

Reply:
@jamiehenson thanks for the info!! i was aiming to be more inline with how we have it in other places ie, https://github.com/ably/docs/blob/main/src/components/Homepage/BodySection/BodySection.tsx, lol and i choose the way i liked the look of :D